shopinvader/tests/test_controller.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.tests.common import (
+    SavepointComponentCase,
+    new_rollbacked_env)
+import requests
+from odoo.addons.component.core import (
+    _get_addon_name,
+)
+
+
+class RegistryMixin(object):
```

Comment:
@sebastienbeau This mixin should go into "base_invader"

Reply:
I think we can move this in base_rest